doc/api/inspector.md

```diff
@@ -108,6 +108,49 @@ with an error. [`session.connect()`] will need to be called to be able to send
 messages again. Reconnected session will lose all inspector state, such as
 enabled agents or configured breakpoints.
 
+## inspector.ws
+
+The Inspector can be interacted with via a websocket-based protocol. These APIs
+allow programatic control of the websocket, equivalent to use `--inspect*` CLI
+options.
+
+### ws.listen()
+
+Start the websocket listener.
+
+### ws.unlisten()
+
+Stop the websocket listener.
+
+### ws.id()
+
+Return the UUID string of the listener, or undefined if the listener is not
+started.
```

Comment:
Maybe you think its a single unchanging value? But it isn't, if you run start.js you'll see 2 or 4 values of this UUID printed. So, it could be a getter, but not a fixed-value property.